deployments/helm-chart/templates/controller-deployment.yaml

```
@@ -56,6 +56,8 @@ spec:
         - name: prometheus
           containerPort: {{ .Values.prometheus.port }}
 {{- end }}
+        resources:
+{{ toYaml .Values.controller.resources | indent 12 }}
```

After https://github.com/nginxinc/kubernetes-ingress/pull/636#discussion_r307244549 this indentation needs to be adjusted
```suggestion
{{ toYaml .Values.controller.resources | indent 10 }}
```

---

Done! I have fixed the indentation.